We have been using nightly features, switching back requries to change
patches again and since its not really supported by them anyways, just
stick to using nightly.
--- a/build/config/rust.gni
+++ b/build/config/rust.gni
@@ -112,7 +113,7 @@
 # `rustc_nightly_capability = false` will make the build avoid the use of
 # Rust nightly features. There are no bots that test this and there is no
 # guarantee it will work, but we accept patches for this configuration.
-rustc_nightly_capability = use_chromium_rust_toolchain
+rustc_nightly_capability = true
 
 # Platform support for the Rust toolchain.
 chromium_toolchain_supports_platform = !is_wasm
